possible process change on security hire

chayim opened the pull request:
The potential process change is highlighted here.  I'm thinking that we can ask new hires to explain a vulnerability, in plain English, and assess it's impact.  This will allow us to both gauge their understanding of security, and their ability to communicate.  To wit, we would be relying on an existing, public vulnerability, already disclosed in the [mitre database](https://cve.mitre.org/), and asking for a two to three paragraph response.

nicksnyder commented on job-descriptions/software-engineer-security.md:
```diff
@@ -38,15 +38,14 @@ Learn more about what it is like to work at Sourcegraph by reading [our handbook
-1. You write your first Sourcegraph [RFC](https://about.sourcegraph.com/handbook/communication/rfcs) by creating a new Google Doc using [this template](https://docs.google.com/document/d/1ol7aVXuXB7XL4DorOoxoDsaSyFI9Pv4Bcc1zfo-iLtw/edit#). We will review your RFC and may ask questions via comments directly on the document.
-1. You speak with the Sourcegraph Cloud engineering manager for 1 hour about your RFC.
-1. We schedule 4 hours of remote interviews over video chat across multiple days.
+1. You write a short security assessment, taking less than **30m** of your time, explaining a security vulnerability.
```
Can you flesh this out more? Is this a security assessment about Sourcegraph or could it be about a past security threat? Can a candidate share something they have already written?

chayim replied:
I'm going to dummy up a template, much like the RFC.  The idea is that a candidate would explain a vulnerability for which we provide the appropriate research.  The candidate will help explain the vulnerability **in plain English**.  Given the information we provide, they would ideally explain the impact of the vulnerability to the Sourcegraph product.  I should have it ready tomorrow afternoon.

nicksnyder commented:
ping on getting this merged

chayim commented:
Today.

On Wed, Aug 19, 2020 at 8:55 PM Nick Snyder <notifications@github.com>
wrote:

> ping on getting this merged
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sourcegraph/careers/pull/115#issuecomment-676572791>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAKFVRVXH3J4AC4GVCXKAYDSBQGYLANCNFSM4PU2JESA>
> .
>

chayim commented:
> ping on getting this merged

Updated with a sample CVE and changed the timeline. I feel like it was unfair to the candidate. Have a look - I'm pro merging, but wanted to run it by you.

nicksnyder commented on job-descriptions/software-engineer-security.md:
```diff
@@ -38,15 +38,14 @@ Learn more about what it is like to work at Sourcegraph by reading [our handbook
-1. You write your first Sourcegraph [RFC](https://about.sourcegraph.com/handbook/communication/rfcs) by creating a new Google Doc using [this template](https://docs.google.com/document/d/1ol7aVXuXB7XL4DorOoxoDsaSyFI9Pv4Bcc1zfo-iLtw/edit#). We will review your RFC and may ask questions via comments directly on the document.
-1. You speak with the Sourcegraph Cloud engineering manager for 1 hour about your RFC.
-1. We schedule 4 hours of remote interviews over video chat across multiple days.
+1. You write a short security assessment, taking less than **1hr** of your time, explaining a security vulnerability [sample problem](https://docs.google.com/document/d/1oXhjU_3y2uhpmWJ2hD0NTaqSanUAOlKDFFmjazAU3Rg/).
```
Please share this doc with the Sourcegraph org so we can suggest/edit/comment. Can you inline instructions into that document?
